### PR TITLE
Release/sim 1610 release 3.8.0 and jmp 2.5.0

### PR DESCRIPTION
**Pull request by luigiwerzowa**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,6 +6,25 @@ title: jSparrow
...
+*A little bird, with plumage brown,  
+On top my codebase flutters down,  
+In search for issues where they may lay,  
+Till, full on bugs, it flies away.*
+
+jSparrow has been busy studying new rules and is therefore happy to announce that he already knows $2^6$ rules! That's right, jSparrow supports [***64 automatic refactoring rules!***](https://jsparrow.github.io/rules/)
+
+### New rules to improve performance and readability!
+
+* [Use Collections Singleton List](/rules/use-collections-singleton-list.html) offers an efficient way to to create empty or single-element lists.
```
Comment (arditymeri) [inline docs/README.md]: to to create...
One to too much :D

Comment (arditymeri) [inline docs/README.md]: I'm afraid we need the full link of the rule here, don't we?

File: docs/README.md [hunk 2]
```diff
@@ -6,6 +6,25 @@ title: jSparrow
...
+On top my codebase flutters down,  
+In search for issues where they may lay,  
+Till, full on bugs, it flies away.*
+
+jSparrow has been busy studying new rules and is therefore happy to announce that he already knows $2^6$ rules! That's right, jSparrow supports [***64 automatic refactoring rules!***](https://jsparrow.github.io/rules/)
+
+### New rules to improve performance and readability!
+
+* [Use Collections Singleton List](/rules/use-collections-singleton-list.html) offers an efficient way to to create empty or single-element lists. 
+* [Remove Null Checks Before Instanceof](/rules/remove-null-check-before-instanceof.html) recognizes redundant null checks before `instanceof` and removes them.
```
Comment (arditymeri) [inline docs/README.md]: Here as well.

File: docs/README.md (hunk 1 above)
Comment (luigiwerzowa) [reply docs/README.md]: Oops, thanks for noticing!

Comment (luigiwerzowa) [reply docs/README.md]: Thanks! Fixed.

**Pull request merged**